### ISSUE TITLE
 Add saundarananda-edmatsunami

### ISSUE THREAD

**Issue by kiranlakkur** [ambuda-org/gretil#5]
As part of expanding texts available on ambuda, clean up saundarananda XMLfile(Filename: sa_azvaghoSa-saundarAnanda-edmatsunami.xml).

**Comment by kiranlakkur** [comment]
Following changes were made as part of cleanup:

- Added `<div>` block to each sarga.

- Added `<head>` block for each sarga.

    `<head>prathamaḥ sargaḥ</head>`

- Added `<trailer>` to third sarga.

    `<trailer>Saundaranande mahākāvye tathāgathavarṇano nāma tṛtīyaḥ sargaḥ |</trailer>`

- Verses 1.3 and 1.4 - Paadas from these verses were mixed up. Corrected it.

   Changed from:
    ```
    <lg>
    <l>haviḥṣu yaś ca svātmārthaṃ gām adhukṣad vasiṣṭhavat /</l>
    <l>tapaḥśiṣṭeṣu ca śiṣyeṣu gām adhukṣad vasiṣṭhavat /</l>
    <l>māhātmyād dīrghatapaso yo dvitīya ivābhavat // Saund_1.3 //</l>
    </lg>
    
    <l>tṛtīya iva yaś cābhūt kāyāṅgirasayor dhiyā // Saund_1.4 //</l>
    ```
  Changed to:
  ```
  <lg xml:id="Saund.1.3">
  <l>haviḥṣu yaś ca svātmārthaṃ gām adhukṣad vasiṣṭhavat |</l>
  <l>tapaḥśiṣṭeṣu ca śiṣyeṣu gām adhukṣad vasiṣṭhavat |</l>
  </lg>
  <lg xml:id="Saund.1.4">
  <l>māhātmyād dīrghatapaso yo dvitīya ivābhavat |</l>
  <l>tṛtīya iva yaś cābhūt kāyāṅgirasayor dhiyā || 4 ||</l>
  </lg>
  ```

- Verses 2.33 and 2.34 - These verses were not wrapped in <lg> tags. Corrected it

    Changed from:
    ```
    <p>rāṣṭram anyatra ca baler na sa kiṅ cid dadīdapat /
    <l>bhṛtyair eva ca sodyogaṃ dviṣaddarpam adīdapat // Saund_2.33 /</l>
    <lg>
    <l>svair evādīdapac cāpi bhūyo bhūyo guṇaiḥ kulam /</l>
    <l>prajā nādīdapac caiva sarvadharmavyavasthayā // Saund_2.34 //</l>
    </lg></p>
    ```
    Changed to:
    ```
    <lg xml:id="Saund.2.33">
    <l>rāṣṭram anyatra ca baler na sa kiṅ cid dadīdapat |</l>
    <l>bhṛtyair eva ca sodyogaṃ dviṣaddarpam adīdapat || 33 ||</l>
    </lg>
    <lg xml:id="Saund.2.34">
    <l>svair evādīdapac cāpi bhūyo bhūyo guṇaiḥ kulam |</l>
    <l>prajā nādīdapac caiva sarvadharmavyavasthayā || 34 ||</l>
    </lg>
    ```

- Verse 3.18 - Verse had a hyphen. Corrected it.

    Changed from:
    `naikavidhabhayakareṣu kim-u svajanasvadeśajanamitravastuṣu`

    Changed to:
    `naikavidhabhayakareṣu kimu svajanasvadeśajanamitravastuṣu`


- Verse 8.48 - Had an additioal paada in the verse. Removed it.

    Changed from:    
    ```
    <l>anulepanam añjanaṃ srajo maṇimuktātapanīyam aṃśukam /</l>
    <l>yad ahany ahani pradhāvanair vasanaiś cābharaṇaiś ca saṃskṛtam /</l>
    <l>aśubhaṃ tamasāvṛtekṣaṇaḥ śubhato gacchasi nāvagacchasi // Saund_8.48 //</l>
    ```
    Changed to:
    ```
    <l>yad ahany ahani pradhāvanair vasanaiś cābharaṇaiś ca saṃskṛtam |</l>
    <l>aśubhaṃ tamasāvṛtekṣaṇaḥ śubhato gacchasi nāvagacchasi || 48 ||</l>
    ```

- Verse 9.23 - Was missing first paada of the verse. Added

  Changed from:
  `<l>tathā matā vīratarā manīṣiṇo jayanti lohāni ṣaśindriyāṇi ye // Saund_9.23 //</l>`    
  
  Changed to:
  ```
  <l>tathā hi vīrāḥ puruṣā na te matā jayanti ye sāśvarathadvipānarīn |</l>
  <l>yathā matā vīratarā manīṣiṇo jayanti lolāni ṣaḍindriyāṇi ye || 23 ||</l></lg>
  ```

- Verses 10.22 and 10.23 - Verses were not wrapped in <lg> tag. Corrected them.

  Changed from:
  ```
  <p>nānāvirāgāṇy atha pāṇḍarāṇi suvarṇabhaktivyavabhāsitāni /
  <l>atāntavāny ekaghanāni yatra sūkṣmāṇi vāsāṃsi phalanti vṛkṣā // Saund_10.22 /</l>
  <lg>
  <l>hārān maṇīn uttamakuṇḍalāni keyūravaryāṇy atha nūpurāṇi /</l>
  <l>evaṃvidhāny ābharaṇāni yatra svargānurūpāṇi phalanti vṛkṣaḥ // Saund_10.23 //</l>
  </lg></p>
  ```
  Changed to:
  ```
  <lg xml:id="Saund.10.22">
  <l>nānāvirāgāṇy atha pāṇḍarāṇi suvarṇabhaktivyavabhāsitāni |</l>
  <l>atāntavāny ekaghanāni yatra sūkṣmāṇi vāsāṃsi phalanti vṛkṣā || 22 ||</l>
  </lg>
  <lg xml:id="Saund.10.23">
  <l>hārān maṇīn uttamakuṇḍalāni keyūravaryāṇy atha nūpurāṇi |</l>
  <l>evaṃvidhāny ābharaṇāni yatra svargānurūpāṇi phalanti vṛkṣaḥ || 23 ||</l>
  </lg>
  ```

- Verse 11.20 - A comma was present in the verse. Corrected it.

  Changed from:
  `auddhatyam atha vaktṝmā, abhidāsyāmi tattvataḥ`
  
  Changed to:
  `auddhatyam atha vaktṝmā abhidāsyāmi tattvataḥ`


- Verse 16.53 - Has text wrapped in brackets

  `evaṃ hi cittaṃ praśamaṃ na yāti [na vahri] nā vahnir iveryamāṇaḥ`
  Left as is

- Verse 18.53 - Was missing first paada pf the verse. Corrected it.

  Changed from:
  `<l>ato'sti bhūyas tvayi me vivakṣitaṃ nato hi bhaktaś ca niyogam arhasi // Saund_18.53 //</l>` 
  
  Changed to:
  ```
  <l>sadharma dharmānvayato yataśca te mayi prasādo'dhigame ca kauśalam |</l>
  <l>ato'sti bhūyas tvayi me vivakṣitaṃ nato hi bhaktaś ca niyogam arhasi || 53 ||</l></lg>
```

**Comment by akprasad** [comment]
sounds great! Feel free to send a pull request directly.

**Comment by kiranlakkur** [comment]
Thanks. I  have a raised a [PR](https://github.com/ambuda-project/gretil/pull/6) now.

**Comment by akprasad** [comment]
Landed in #6
